cypress/features/regression/experimental/groupedCharacter.feature

```diff
@@ -1,9 +1,9 @@
-Feature: Experimental GroupedCharacter component
-  I want to check Experimental Grouped character component properties
+Feature: GroupedCharacter component
+  I want to check Grouped character component properties
 
   @positive
   Scenario Outline: Set groups to <nameOfObject> and verify input
-    Given I open default "Experimental GroupedCharacter Test" component in noIFrame with "groupedCharacter" json from "experimental" using "<nameOfObject>" object name
+    Given I open default "GroupedCharacter Test" component in noIFrame with "groupedCharacter" json from "experimental" using "<nameOfObject>" object name
```

Comment:
We should also move the `cypress/fixtures/experimental/groupedCharacter.json` file into `cypress/fixtures/commonComponents` and change all import below

```suggestion
    Given I open default "GroupedCharacter Test" component in noIFrame with "groupedCharacter" json from "commonComponents" using "<nameOfObject>" object name
```